contributing/DEVELOPING.md

```diff
@@ -35,7 +35,6 @@ Before you start writing code, consult these resources for guidance and guidelin
 - **CSS**: [Writing CSS For AMP Runtime](writing-css.md)
 - **Creating new components**:
   - [Instructions and Guidelines for building an AMP component](building-an-amp-extension.md)
-  - Learn to create your first component in this [codelab](https://codelabs.developers.google.com/codelabs/creating-your-first-amp-component/#0)
```

Comment:
Why is this removed instead of updated as it is in `contributing-code.md`?

Reply:
I was under the impression the preferred link was `building-an-amp-extension.md`, which is already being mentioned in the line above. Does that work?

Reply:
Ah yes, thanks.